### DISCUSSION TITLE
Fermata auto automatic play on youtube

### DISCUSSION THREAD

**Discussion by MicheleFire2022** [AndreyPavlenko/Fermata#260] [Q&A]
Hi Guys,
I tried searching but couldn't find any solution so far. When I open youtube in FermataAuto after a few seconds he automatically starts the video he wants. Is there a way to not have it play anything automatically and stay fixed on the youtube home?
thanks for support

Bye
Michele

**Comment by eduxpress** [comment]
First, flip your phone horizontally to stop the video from going full screen.
Now click on the account icon on the top right corner and go to "Settings".
Now open the "Account" menu and uncheck the box that says "Video previews" or something similar to that.
There you go, YouTube will now longer play random videos automatically everytime you open it.
